tests/test_opds_import.py

```diff
@@ -1588,15 +1588,14 @@ def test_resources_are_mirrored_on_import(self):
         imported_representations = [
             e1_oa_link.resource.representation,
             e1_image_link.resource.representation,
-            e1_image_link.resource.representation.thumbnails[0],
```

Comment:
Changes below this point are important.

Reply:
You should update the comment above this since the things that were mirrored changed.

Reply:
And also make sure we have another test that checks that a thumbnail is generated and mirrored when the original is PNG, since that was removed here.